rules/library.bzl

```diff
@@ -345,14 +345,13 @@ def _xcframework(*, library_name, name, slices):
                 elif arch == "arm64":
                     if platform_variant == "simulator":
                         arm64_simulator_slice = name
-
-                        # Skip this - it's later defined
```

Comment:
@jerrymarino you added this originally, do you have any background context or know what might break that isn't covered by the tests?

Reply:
There were some very odd permutations of xcframeworks we handle and can try to run this code against with CI, it's unlikely all possible permutations are tested here 

AFAIK it seems to create an alias to this [here](https://github.com/bazel-ios/rules_ios/pull/810/files#diff-23068963449a2bf4d162d813a4dda51236b9d044d1442e8a4bb4ad7680473478R382)